apiserver/facades/client/bundle/bundle.go

```diff
@@ -96,3 +122,23 @@ func (b *bundleAPI) GetChanges(args params.BundleChangesParams) (params.BundleCh
 	}
 	return results, nil
 }
+
+// ExportBundle exports the current model configuration as bundle.
+func (b *BundleAPI) ExportBundle() (params.StringResult, error) {
+	if err := b.checkCanRead(); err != nil {
+		return params.StringResult{}, errors.Trace(err)
```

Comment:
For errors returned from top level facade functions, use common.ServerError(err)

Reply:
done